src/pages/projects.js

```diff
@@ -775,7 +775,34 @@ const ProjectDescriptions = () => (
         </div>
       </div>
     </div>
+
+    <div className="project-box">
+      <TitleWithAnchor headerClassName="project-major-title projects-common-title" headerTag="h3">eCapture</TitleWithAnchor>
+      <div className="project-minor-title">SSL/TLS capture tool using eBPF</div>
+      <div className="project-body">
+        <a
+            className="project-logo"
+            target="_blank"
+            href="https://github.com/ehids/ecapture"
+        >
+          <img src={require("../assets/projects-logos/ecapture-logo.png")} alt="ehids/eCapture project"/>
+        </a>
+        <div className="project-description">
+          <p>
+            <a href="https://github.com/ehids/ecapture" target="_blank">
+              <b>GitHub</b>
+            </a>{" "}
+          </p>
+          <p>
+            eCapture is a tool that can capture plaintext without a CA certificate. Supports TLS encryption libraries such as openssl/gnutls/nspr etc. The userspace code is written in GO by cilium/ebpf. It can works above Linux Kernel 4.18, and supports CO-RE features.
```

Comment:
```suggestion
            eCapture is a tool that can capture plaintext without a CA certificate. It supports TLS encryption libraries such as openssl/gnutls/nspr etc. The userspace code is written in Go and uses cilium/ebpf. It can work on Linux Kernel 4.18 and later, and supports CO-RE features.
```

Reply:
Got it. thanks.